packages/gr-sheet/src/index.ts

```diff
@@ -717,7 +717,7 @@ const SupportedSheets = {
         extentRef,
         coordinateReferenceEpoch: item.coordinateReferenceEpoch.trim() || null,
       } as const;
-      if (item.type === 'GeodeticDatum') {
+      if (/geodetic(?: *datum *)?$/i.test(item.type)) {
```

Comment:
I mean, if it works it works. To be honest this application of regexes triggers me, I suppose it could be simpler, but I won’t request this to be changed (not to mess with black magic that regexes are)